code/modules/mob/living/simple_animal/slime/life.dm

```diff
@@ -112,7 +112,7 @@
 
 	var/loc_temp = get_temperature(environment)
 
-	bodytemperature += adjust_body_temperature(bodytemperature, loc_temp, 1)
+	adjust_bodytemperature(adjust_body_temperature(bodytemperature, loc_temp, 1))
```

Comment:
this is the future you chose

Reply:
It's just one spot it'll be ok someone will make it prettier later :)